### ISSUE TITLE
Figure out why the Python 3.5 travis build takes over 50% longer than Python 3.2

### ISSUE THREAD

**Issue by JukkaL** [python/mypy#911]
For example, see https://travis-ci.org/JukkaL/mypy/builds/84849832

I'd hope that Python 3.5 isn't that much slower.

**Comment by o11c** [comment]
Note how it's a download, not the same source as the others, maybe it's not compiled with the same level of optimization?

**Comment by JukkaL** [comment]
Okay that might explain it. Anybody care to build Python 3.5 locally and benchmark against Python 3.4?

**Comment by o11c** [comment]
I have everything from 3.2 to 3.5 installed locally for test purposes (from Debian packages, though some are removed from the archive so you have to get them from snapshots.d.o)

Testing just pythoneval, 3.5 is 14% slower, not nearly as drastic as on travis.

```
python3.4 ./runtests.py eval  228.43s user 8.99s system 99% cpu 3:58.29 total
python3.5 ./runtests.py eval  259.82s user 8.98s system 99% cpu 4:29.79 total
```

**Comment by ethanhs** [comment]
This can be closed, 3.5 is also now one of the fastest on Travis. 😃
